runtime/j9vm/exports.cmake

```diff
@@ -330,26 +331,22 @@ if(NOT JAVA_SPEC_VERSION LESS 11)
 		JVM_HasReferencePendingList
 		JVM_WaitForReferencePendingList
 
-		#Additions for Java 9 (General)
+		# Additions for Java 9 (General)
 		_JVM_GetNanoTimeAdjustment@16
 
-		#Additions for Java 11 (General)
+		# Additions for Java 11 (General)
 		JVM_BeforeHalt
 		JVM_GetNestHost
 		JVM_GetNestMembers
 		JVM_AreNestMates
 		JVM_InitClassName
 		JVM_InitializeFromArchive
 	)
-
-else()
-	# ie. JAVA_SPEC_VERSION <= 10
-	jvm_add_exports(jvm _JVM_GetCallerClass@4)
 endif()
 
 if(NOT JAVA_SPEC_VERSION LESS 14)
```

Comment:
No `>=` operator in cmake?

Reply:
The version of cmake we set as a minimum (3.4) does not support `<=` or `>=`, so we say `not less` instead of `>=`; support for those operators was added in 3.7 [1].

[1] https://cmake.org/cmake/help/v3.14/release/3.7.html?highlight=LESS_EQUAL